Detections/QualysVM/NewHighSeverityVulnDetectedAcrossMulitpleHosts.yaml

```diff
@@ -0,0 +1,23 @@
+id: 84cf1d59-f620-4fee-b569-68daf7008b7b
+name: New High Severity Vulnerability Detected Across Multiple Hosts
+description: |
+  'This creates an incident when a new high severity vulnerability is detected across multilple hosts'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CustomConnector
```

Comment:
Same as previous for connectorId and dataTypes.

Reply:
corrections have been made and pushed.